{
  myRepository.setPassword("wrong-password");
  try {
    myRepository.testConnection();
    fail("testConnection() should fails, when wrong credentials specified");
  }
 catch (  Exception e) {
    assertEquals(TaskBundle.message("failure.login"),e.getMessage());
  }
}
